packages/nuxt/src/pages/runtime/utils.ts

```diff
@@ -25,6 +25,10 @@ export const wrapInKeepAlive = (props: any, children: any) => {
   return { default: () => import.meta.client && props ? h(KeepAlive, props === true ? {} : props, children) : children }
 }
 
+export const wrapInTransition = (props: any, children: any) => {
+  return { default: () => import.meta.client && props ? h(Transition, props === true ? {} : props, children) : children.default?.() }
+}
```

Comment:
because this utility function is needed by nuxt core (not just the pages module), we should probably keep it in `packages/nuxt/src/app/components/utils.ts` (unless there's a reason I haven't spotted that it needs to be moved)